orchestrate/dags/other_examples/blue_green_dag.py

```diff
@@ -1,31 +1,29 @@
 """## Datacoves Blue-Green DAG"""
 
-from airflow.decorators import dag
-from operators.datacoves.dbt import DatacovesDbtOperator
+from airflow.decorators import dag, task
 from pendulum import datetime
 
-
 @dag(
-    doc_md = __doc__,
+    doc_md=__doc__,
     default_args={
         "start_date": datetime(2024, 1, 1),
         "owner": "Datacoves",
         "email": "bruno@example.com",
         "email_on_failure": True,
         "retries": 1,
     },
-    catchup = False,
-    tags = ["transform"],
-    description = "Datacoves blue-green run",
-    schedule = "@daily",
+    catchup=False,
+    tags=["transform"],
+    description="Datacoves blue-green run",
+    schedule="@daily",
 )
 def datacoves_bluegreen_dag():
-    blue_green_run = DatacovesDbtOperator(
-        task_id="blue_green_run",
-        bash_command="dbt-coves blue-green --dbt-selector '-s personal_loans'",
-    )
-    blue_green_run
 
+    @task.datacoves_dbt(connection_id="main") 
+    def blue_green_run():
+        return "dbt-coves blue-green --dbt-selector '-s personal_loans'"
```

Comment:
@BAntonellini if I now use connection_id, how would this get DATACOVES__MAIN__DATABASE?  do we need to change this command to also use a connection_id?